opensearch-ruby.gemspec

```diff
@@ -66,5 +65,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.5'
 
   s.add_dependency 'faraday', '>= 1.0', '< 3'
-  s.add_dependency 'multi_json'
+  s.add_dependency 'multi_json', '~> 1.0'
```

Comment:
This is suspicious, did you mean `>= 1.0`? They are at 1.15. There's no reason why a 2.x won't work when that comes out.

Reply:
It was intentional because I didn't want `2.x` breaking changes to affect us. But, I guess then some of our specs would fail as a result as we can handle this issue then. Going to update it per your suggestion.

Reply:
Yep, generally you lock down semver a minimum of what's known at the time of writing. Otherwise you end up with an impossible to resolve matrix for no reason.